service/create/service.go

```diff
@@ -462,7 +463,7 @@ func validateIDs(ids []string) bool {
 
 func (s *Service) addFunc(obj interface{}) {
 	cluster := *obj.(*awstpr.CustomObject)
-	s.logger.Log("info", fmt.Sprintf("creating cluster '%s'", cluster.Name))
+	s.logger.Log("info", fmt.Sprintf("creating cluster '%s'", key.ClusterID(cluster)))
```

Comment:
@kopiczko this does it only per function call. You would need to do this everywhere anyway. What I am thinking is to have a log resource which does logging according to information in a context. We could have `micrologger.Logger.WithContext(ctx, key, val)` and add the key value pairs to the context at the very beginning of the reconciliation loop initialization. The log resource then takes the key-value pairs from the context and adds them to the logging it does. We would then also need to extend the framework interface to take the context.